lua/dashboard/init.lua

```diff
@@ -39,9 +39,11 @@ function M.setup(opts)
 end
 
 function M.instance()
+    vim.api.nvim_create_augroup("dashboard_open", {clear = true})
     local bufnr = ui.create_buffer()
     ui.load(bufnr)
     vim.api.nvim_create_autocmd('VimResized', {
```

Comment:
Hi there, good catch!
I never saw that error before but definitely pops up in the messages.

I think this works, but making the autocommand buffer local would be my preference since I won't need to worry about clearing the autocommand in another spot in the code. Would look like:

```lua
vim.api.nvim_create_autocmd('VimResized', {
    buffer = bufnr,
    callback = function()
        ui.load(bufnr)
    end,
})
```

Here's the help doc info about setting the buffer option:

```
buffer (integer) optional: buffer number for buffer-local
autocommands |autocmd-buflocal|. Cannot be used with
{pattern}.
```